Sniffs/Classes/InstantiateNewClassesSniff.php

```diff
@@ -101,7 +101,7 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         if( ! $valid)
         {
             $error = 'Instanciating new classes without parameters does not require brackets.';
-            $phpcsFile->addError($error, $stackPtr, 'New class');
+            $phpcsFile->addFixableError($error, $stackPtr, 'New class');
```

Comment:
If you don't actually provide any fixing code, then reporting error/warning as fixable would fail coding standard tests. You don't have any tests in the standard and that's why you don't see the difference.